ivy/functional/ivy/losses.py

```diff
@@ -156,7 +156,7 @@ def binary_cross_entropy(
     """
     pred = ivy.clip(pred, epsilon, 1 - epsilon)
     return ivy.negative(
-        ivy.add(ivy.log(pred) * true, ivy.log(1 - pred) * (1 - true), out=out), out=out
+        ivy.add(ivy.log(pred) * true, ivy.log(1 - pred) * (1 - true)), out=out
```

Comment:
we should use the `out` argument for both the `add` function and also the `negative` function. These functions are called sequentially, and so the same memory can be used for storing the result of both operations.